src/borg/testsuite/helpers.py

```diff
@@ -904,6 +904,8 @@ def test_yes_env_output(capfd, monkeypatch):
 
 
 def test_progress_percentage_sameline(capfd):
+    os.environ['COLUMNS'] = '4'
+    os.environ['LINES'] = '1'
```

Comment:
ok, so if what you want is a temporary change of these env vars for this test, look at other places in the src where monkeypatch is used to do that.